### DISCUSSION TITLE
Frigate webUI in other languages?

### DISCUSSION THREAD

**Discussion by lukasig** [blakeblackshear/frigate#4784] [Ideas]
Hi

I was just wondering if we could expect the possibility of translating the webUI in languages other than english. Can the project make use of tools such as weblate.org?

I would be happy to be able to contribute and I think others will be as well.

Even if it's not coming any time soon, I must say that this is an amazing project. All due respect to @blakeblackshear and the others who are doing amazing work.

**Comment by NickM-27** [comment]
I have a draft PR at https://github.com/blakeblackshear/frigate/pull/3718 although it has fallen behind a bit as I've been working on the current release (and it won't be ready for that) but hopefully after 0.12 is fully released I can get that fully merged and create docs for adding other languages
